APIManager.js

```diff
@@ -0,0 +1,103 @@
+class user {
+    constructor(firstName, lastName) {
+        this.first = firstName
+        this.last = lastName
+    }
+}
+class mainUser extends user {
+    constructor(firstName, lastName, city, state, imgUrl) {
+        super(firstName, lastName)
+        this.city = city
+        this.state = state
+        this.imgUrl = imgUrl
+    }
+}
+
+class APIManager {
+    constructor() {
+        this.data = {}
+        this.savedUsers = []
+    }
+
+    createUsers(respone) {
+        const results = respone.results
+        const firstUser = results[0]
+        this.data.mainUser = new mainUser(
+             firstUser.name.first,
+             firstUser.name.last,
+             firstUser.location.city, 
+             firstUser.location.state, 
+             firstUser.picture.large)
+        results.shift()
+        this.data.usersArray = results.map(u => {
+            return new user(u.name.first, u.name.last)
+        });
+
+
+    }
+
+    userAPIRequests() {
```

Comment:
The convention is to call the methods that make API requests (GET) as `getUser` or `fetchUser`